src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -115,6 +117,13 @@ export default function connectHitsPerPage(renderFn) {
     }
 
     return {
+      getConfiguration() {
+        const { value: defaultHitsPerPage } =
+          items.find(item => item.default) || items[0];
```

Comment:
This changes the previous behaviour (not default set). By default, we were using the hits per page used by the engine and configured in the index. We should not change that and therefore we should not set any value if there are no `default: true`.

Reply:
:+1: